### DISCUSSION TITLE
New Blueprint: Dim Screen with Sun

### DISCUSSION THREAD

**Discussion by aderusha** [HASwitchPlate/HASPone#124] [Show and tell]
I'd been using a 3rd party blueprint for this but kept having problems with reliability that I was never able to track down.

So instead I went and built a HASPone-specific blueprint which you can grab here:

[![Dim Screen with Sun](https://my.home-assistant.io/badges/blueprint_import.svg)](https://my.home-assistant.io/redirect/blueprint_import/?blueprint_url=https%3A%2F%2Fgithub.com%2FHASwitchPlate%2FHASPone%2Fblob%2Fmain%2FHome_Assistant%2Fblueprints%2Fhasp_Dim_Screen_with_Sun.yaml)

I have a HASPone in my bedroom which gets a lot of sunlight.  During the day I want the backlight at full brightness, but if I'm ever going to get any sleep I want that dialed way down at night.  The default values set in this blueprint behave in this mode.

You can also reverse the daytime/nightime brightness values to have HASPone act as a night light, brightening at night and then dim during the day.  I'm not here to tell you what to do, have fun go bananas :D

**Comment by Petervanoos** [comment]
Nice. Thx!
